src/components/CalculatorСalorieForm/CalorieForm.jsx

```diff
@@ -0,0 +1,117 @@
+import { React, useState } from 'react';
+import { Grid, Radio, RadioGroup, FormControl } from '@mui/material';
+import { useForm } from 'react-hook-form';
```

Comment:
we are using useFormik for all forms

Reply:
Why I not informed? I use not only validation on submit, I use validation on input and submit. Can I do it with formik? And why I **can** use react-hook-form and submit work? Good question?

Reply:
Our mentor told us to use "formik" and our team used it.

Reply:
I added formik